PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs

```diff
@@ -101,8 +101,17 @@ public static class CatchNearbyPokemonsTask
                     }
                 }
             }
-            
-            if (0 < pokemons.Count)
+
+            var allitems = await session.Inventory.GetItems().ConfigureAwait(false);
+            var pokeBallsCount = allitems.FirstOrDefault(i => i.ItemId == ItemId.ItemPokeBall)?.Count;
+            var greatBallsCount = allitems.FirstOrDefault(i => i.ItemId == ItemId.ItemGreatBall)?.Count;
+            var ultraBallsCount = allitems.FirstOrDefault(i => i.ItemId == ItemId.ItemUltraBall)?.Count;
+            var masterBallsCount = allitems.FirstOrDefault(i => i.ItemId == ItemId.ItemMasterBall)?.Count;
+            masterBallsCount =
```

Comment:
If i can remember, i did a calling similar to this during SuperBuild, and it gave off errors in build.bat. If it runs successfully in build.bat, this comment can be ignored.